website/docs/r/federated_settings_org_config.html.markdown

```diff
@@ -34,6 +35,8 @@ data "mongodbatlas_federated_settings_org_configs" "org_configs_ds" {
 * `federation_settings_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration. 
 * `org_id` - (Required) Unique 24-hexadecimal digit string that identifies the organization that contains your projects.
 * `domain_allow_list` - List that contains the approved domains from which organization users can log in.
+* `post_auth_role_grants` - (Optional) List that contains the default [roles](https://www.mongodb.com/docs/atlas/reference/user-roles/#std-label-organization-roles) granted to users who authenticate through the IdP in a connected organization. If you provide a postAuthRoleGrants field in the request, the array that you provide replaces the current postAuthRoleGrants.
```

Comment:
Not blocking but the 2nd half of this doesn't make a great deal of sense to a Terraform user, i.e. "If you provide a postAuthRoleGrants field in the request, the array that you provide replaces the current postAuthRoleGrants."

Reply:
we can update with "List that contains the default roles granted to users who authenticate through the IdP in a connected organization. If you provide a postAuthRoleGrants field in the request, the array that you provide replaces the current postAuthRoleGrants." 

source - https://www.mongodb.com/docs/atlas/reference/api/org-mapping-update-one/#request-body-parameters:~:text=postAuthRoleGrants,current%20postAuthRoleGrants.